Tests/Microsoft.AppCenter.Test.Windows/Storage/FakeStorageTest.cs

```diff
@@ -85,7 +85,7 @@ public void GetLogsQueryError()
             var mockAdapter = new Mock<IStorageAdapter>();
             mockAdapter.Setup(
                     a => a.GetAsync(It.IsAny<PredType>(), It.IsAny<int>()))
-                .Throws(new StorageException());
+                .ThrowsAsync(new StorageException());
```

Comment:
`ThrowsAsync()` allows to setup mock that returns faulted task. `Throws()` throws right away without returning a task.